code/game/gamemodes/vampire/vampire.dm

```diff
@@ -141,15 +148,21 @@
 				text += "<br><font color='red'><B>The [special_role_text] has failed!</B></font>"
 				feedback_add_details("traitor_success","FAIL")
 
-		world << text
-	return 1
+	return text
```

Comment:
Dead vamps don't have their icon's .Turn('ed, and pretty much every other gamemode has text += "<br><hr>" before returning the text.